jujugui/static/gui/src/app/test-init.js

```diff
@@ -1242,6 +1243,11 @@ describe('init', () => {
         const args = app.state.changeState.args[0];
         assert.equal(args.length, 1, 'changeState num args');
         assert.deepEqual(args[0], {root: null}, 'changeState args');
+        // We are testing this here because this gets called when successfully
+        // logging into a controller and this is typically done from an anonymous
+        // state, even if the user is logging in on the initial load, the app
+        // execution is similar.
+        assert.equal(app.reloadConfigFile.callCount, 1);
```

Comment:
Unfortunately this seems like the best we can do without going full uitest route here as it requires so many moving parts to truly test that this functions properly